<head>
  <meta charset="utf-8">
  <title>Engineering | HUST-China igem 2023</title>
  <meta content="width=device-width, initial-scale=1" name="viewport">
  <meta content="Webflow" name="generator">
  <link href="static/normalize.css" rel="stylesheet" type="text/css">
  <link href="static/webflow.css" rel="stylesheet" type="text/css">
  <link href="static/igem-hust-china-2023.other_pages.css" rel="stylesheet" type="text/css">
  <link href="https://fonts.googleapis.com" rel="preconnect">
  <link rel="license" href="https://creativecommons.org/licenses/by/4.0/" />
  <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin="anonymous">
  <script src="static/jquery.js" type="text/javascript"></script>
  <script src="static/webfont.js" type="text/javascript"></script>
  <script src="static/webflow.js" type="text/javascript"></script>
  <script type="text/javascript">
    WebFont.load({
      google: {
        families: ["Inconsolata:400,700", "Montserrat:100,100italic,200,200italic,300,300italic,400,400italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic", "Droid Serif:400,400italic,700,700italic", "Oswald:200,300,400,500,600,700", "Merriweather:300,300italic,400,400italic,700,700italic,900,900italic"]
      }
    });
  </script>
  <script type="text/javascript">
    ! function (o, c) {
      var n = c.documentElement,
        t = " w-mod-";
      n.className += t + "js", ("ontouchstart" in o || o.DocumentTouch && c instanceof DocumentTouch) && (n.className += t + "touch")
    }(window, document);
  </script>
  <link href="images/favicon.ico" rel="shortcut icon" type="image/x-icon">
  <link href="images/webclip.png" rel="apple-touch-icon">


<style>
    #subtitle1:target {
        padding-top: 60px;
        margin-top: -60px;
    }


    #to-the-top {
        /*svg导航*/
        height: 30px;
        width: 30px;
        bottom: 150px;
        right: 0px;
        position: fixed;
        display: none;
        /*padding: 10px;*/
        margin: 10px;
    }

    #to-the-bottom {
        height: 30px;
        width: 30px;
        bottom: 110px;
        right: 0px;
        position: fixed;
        /*padding: 10px;*/
        margin: 10px;
    }
    body{
      margin=0;
    }
</style>

<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="https://static.igem.wiki/common/icons/favicons/igem-2022.svg"/>
    <link rel="license" href="https://creativecommons.org/licenses/by/4.0/"/>

    <!-- Bootstrap CSS -->
    <link href="static/bootstrap.min.css" rel="stylesheet">

    <!-- JQuery Js -->
    <script type="text/javascript" src="static/jquery.js")}}" ></script>
    <script type="text/javascript" src="static/jqueryrotate.js")}}" ></script>


    <!-- Custom CSS -->
    <link href="static/style.css" rel="stylesheet">

    <!-- Font style-->
    
    <title> | HUST-China - iGEM 2022</title>
    <style>
      @font-face {
	font-family:Silkscreen;
	src: url(static/Silkscreen-Regular.ttf);
}
@font-face {
    font-family: Montserrat;
    src: url(static/Montserrat-Regular.ttf);
}

@font-face {
    font-family: Mukta;
    src: url(static/Mukta-Regular.ttf);
}

@font-face {
    font-family: Ubuntu;
    src: url(static/Ubuntu-Regular.ttf);
}

@font-face {
    font-family: Josefin_Sans;
    src: url(static/JosefinSans-Regular.ttf);
}

      </style>
    
  </head>
<body class="body">
<div data-animation="default" class="navbar-logo-left-container shadow-three w-nav" data-easing2="ease-out"
    data-easing="ease-in" data-collapse="all" role="banner" data-no-scroll="1" data-duration="400" id="page-header"
    data-doc-height="1">
    <div class="navbar-wrapper">
     <div class="w-layout-blockcontainer container-2 w-container">
      <img src="https://static.igem.wiki/teams/4595/wiki/home/logo.png" loading="lazy" alt="" class="team-logo"></div>

      <nav role="navigation" class="nav-menu-wrapper w-nav-menu">
        <ul role="list" class="nav-menu-two w-list-unstyled">
          <li>
            <a href="index.html" target="_self" aria-current="page" class="nav-link w--current">Home</a>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1850" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1851" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div class="text-block-7"><span>Project</span></div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="{{ url_for('pages', page='description') }}"
                  class="nav-dropdown-link w-dropdown-link">Description</a>
                <a href="{{ url_for('pages', page='safety') }}" class="nav-dropdown-link w-dropdown-link">Safety</a>
                 <a href="implementation"
                  class="nav-dropdown-link w-dropdown-link">Proposed<br>Implementation</a>

              </nav>
            </div>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="f73ff7f3-a811-76cc-69d5-a6519c08fec2" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="f73ff7f3-a811-76cc-69d5-a6519c08fec3" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div class="text-block-7">Wet Lab</div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="{{ url_for('pages', page='design') }}" class="nav-dropdown-link w-dropdown-link">Design</a>
                <a href="{{ url_for('pages', page='engineering') }}"
                  class="nav-dropdown-link w-dropdown-link">Engineering</a>
                <a href="{{ url_for('pages', page='protocol') }}" class="nav-dropdown-link w-dropdown-link">Protocol</a>
                <a href="{{ url_for('pages', page='results') }}" class="nav-dropdown-link w-dropdown-link">Results</a>
                <a href="{{ url_for('pages', page='notebook') }}" class="nav-dropdown-link w-dropdown-link">Notebook</a>
                <a href="{{ url_for('pages', page='parts') }}" class="nav-dropdown-link w-dropdown-link">Parts</a>
              </nav>
            </div>
          </li>
          <li>
            <a href="{{ url_for('pages', page='hardware') }}" class="nav-link">Hardware</a>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1869" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f186a" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div>HP</div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="{{ url_for('pages', page='human-practices') }}"
                  class="nav-dropdown-link w-dropdown-link">Human<br>Practices</a>
                <a href="{{ url_for('pages', page='education') }}"
                  class="nav-dropdown-link w-dropdown-link">Education&amp;<br>Communication</a>
              </nav>
            </div>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1845" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1846" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div>Team</div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="{{ url_for('pages', page='contribution') }}"
                  class="nav-dropdown-link w-dropdown-link">Contribution</a>
                <a href="{{ url_for('pages', page='members') }}" class="nav-dropdown-link w-dropdown-link">Members</a>
                <a href="{{ url_for('pages', page='collaboration') }}"
                  class="nav-dropdown-link w-dropdown-link">Collaboration</a>
                <a href="{{ url_for('pages', page='partnership') }}"
                  class="nav-dropdown-link w-dropdown-link">Partnership</a>
                <a href="{{ url_for('pages', page='attributions') }}"
                  class="nav-dropdown-link w-dropdown-link">Attributions</a>
              </nav>
            </div>
          </li>
        </ul>
      </nav>
      <div class="menu-button w-nav-button">
        <div class="icon w-icon-nav-menu"></div>
      </div>
    </div>
</div>

  
  <!-- Header -->
    
<header>

<a href="#sectitle1"><img src="https://2022.igem.wiki/hust-china/static/arrow-bar-down.svg" alt="" id="herfsvg"></a>
<a href="#sectitle1">
  <div id="titlebg" style="background: url(https://static.igem.wiki/teams/4595/wiki/engineering/engineering/enginering-success.jpg); background-size:cover"></div>
  </a>
</header>

  <!-- Page Content -->
  <div class="container">
    
 <div class="container">
        <div class="catalogue">
            <h4>content</h4>
            <ul>
                <li><a href="#sectitle1">Introduction</a></li>
                <li><a href="#sectitle2">CYCLE 1: constructed by homologous recombination method</a></li>
                <li><a href="#sectitle3">CYCLE 2: constructed by double enzyme digestion and enzyme ligation method</a></li>
                 <li><a href="#sectitle4">CYCLE 3: Expression validation</a></li>
            </ul>
        </div>

    <div class="main content">
        <div class="row">
            <div class="col-lg-1"></div>
            <div class="content col-lg-10">
                <article>
                    <h1 class="sectitle title-cover" id="sectitle1">Introduction</h1>
                        <p>We hope to build a co-culture system of <i>Synechocystis</i> PCC6803 and <i>S.oneidensis</i> MR-1, which can use solar energy to fix carbon dioxide to generate electricity. Therefore, we designed several components and four system paths to achieve this. In the build phase, we first used homologous recombination to build these pathways but we encountered some problems. Through continuous trials and summaries, we switched to using the double enzyme digestion and enzyme ligation to construct our pathway, and finally achieved certain results. Our experiments would provide some experience for subsequent teams.</p>
                    <h1 class="sectitle title-cover" id="sectitle2">CYCLE 1: constructed by homologous recombination method</h1>
                    <h2 class="subtitle" id="list1">1.1 Design</h2>
                        <p>To construct a well-functioning system pathway, we hope to utilize four plasmids to improve the efficiency of the electricity production of Shewanella, enhance the intensity of cyanobacteria photosynthesis, and increase the antioxidant activity of cyanobacteria and the stress resistance of Shewanella respectively. Based on existing parts with independent functions, we achieve our goal by connecting multiple components to complete plasmids to form a fully functional system pathway.</p>
                        <p>We wanted to build our plasmid by using homologous recombination. Therefore, we introduce homologous arms into the components when building individual elements, which is the basis of the entire pathways construction.</p>
                        <p>First, In the pLactate pathway, we begin by incorporating the first homologous arm downstream of the slr0168 vector fragment, which includes the ampicillin-resistant gene, and upstream of ldhA-lldP-omcs. Following that, a second homologous arm is introduced downstream of ldhA-lldP-omcs and upstream of the slr0168 vector fragment.</p>
                        <p>In the pResistance6803 pathway, we add the first homologous arm  to the downstream of the slr2030 vector fragment containing the ampicin and kana resistant genes and upstream of dsup-gshA-gshB. A second homology arm is then added downstream of dsup-gshA-gshB and upstream of the slr2030 vector fragment.</p>
                        <p>In the pResistance MR-1 pathway, we add the first homologous arm to the downstream of the PYYDT vector fragment containing the kana resistance gene and upstream of dsup-sodA-oprf. A second homology arm is then added downstream of dsup-sodA-oprf and upstream of the PYYDT vector fragment.</p>
                        <p>In the pElectricty pathway, we add the first homologous arm to the downstream of the PYYDT vector fragment containing the kanamycin resistance gene and upstream of ycel-pncB. A second homology arm was added downstream of ycel-pncB and upstream of nadD-nadE-nadM, and a third homology arm was added downstream of nadD-nadE-nadM and upstream of the PYYDT vector fragment. Homologous arms are added to different gene fragments by PCR using different primers. In principle, these specific homologous arms ensure our experimental feasibility.</p>
                        <img src="https://static.igem.wiki/teams/4595/wiki/engineering/engineering/1.png" alt="" class="normimg1" >
                        <i class="imgexplain">Fig 1. The diagram of constructing the plasmid inserted with pLactate by homologous recombination</i>
                        <img src="https://static.igem.wiki/teams/4595/wiki/engineering/engineering/presistance6803.png" alt="" class="normimg1">
                        <i class="imgexplain">Fig 2. The diagram of constructing the plasmid inserted with pResistance6803 by homologous recombination</i>
                        <p></p>
                    <h2 class="subtitle" id="list2">   
                        1.2 Build
                    </h2>   
                        <p>Based on the principle of seamless cloning, we tried to use homologous recombinase to attach multiple fragments directly into the vector to form a complete plasmid. </p> 
                        <p>Take the pElectricty system pathway as an example: first, the linearized vector PYYDT and gene fragments - ycel-pncB and nadD-nadE-nadM were obtained by PCR. They were then validated and purified by gel extraction. Finally, homologous recombinase was used to directly connect the linearized vector PYYDT with two fragments of ycel-pncB and nadD-nadE-nadM to construct a complete system pathway. We then transfered the homologous recombinant product into <i>E. coli</i> TOP10 and seed into LB plates containing 50 μg/mL kanamycin for screening. When single colonies appeared on the surface of solid media, we singled out the single colonies for PCR validation. After observing the correct bands, we mixed 800 μl of bacterial solution with 800 μl of 1:1 mixture of glycerol and double distilled water and keep the seeds at -80℃.</p>
                    <h2 class="subtitle" id="list3">   
                        1.3 Result
                    </h2>   
                        <p>We tried to construct our pathway through multi-fragment homologous recombination. After several attempts, we successfully constructed plasmids omcs, ycel-pncB and nadE-nadD-nadM.</p>  
                        <img src="https://static.igem.wiki/teams/4595/wiki/engineering/engineering/tycz.png" alt="" class="normimg1">
                        <i class="imgexplain">Fig 3. The left figure is the PCR identification result of omcs plasmid.It shows that there is a bright band at about 2000 bp, and the length of omcs plasmid is 1867 bp, which proves that we have successfully constructed the plasmid of omcs system pathway.</i>
                        <i class="imgexplain">The right figure is the PCR identification result of ycel-pncB plasmid and nadE-nadD-nadM plasmid.It shows that there is a bright band at about 2500 bp and a bright band at about 3000 bp. The length of ycel-pncB plasmid is 2601 bp and the length of nadE-nadD-nadM plasmid is 2820 bp , which proves that we have successfully constructed the plasmid of ycel-pncB system pathway and the plasmid of nadE-nadD-nadM system pathway.</i>
                        <p></p>
                    <h2 class="subtitle" id="list4">   
                        1.4 Experience
                    </h2>  
                        <p>After several attempts, we found that the direct use of multi-fragment homologous recombination is not suitable for most pathways, so we carefully analyzed and summarized the reasons for the failures. We believe that the success rate of homologous recombination is low due to the large number of genes, the huge number of bases and the complex topology contained in the pResistance MR-1, pLactate, pResistance 6803, pElectricity pathways. Therefore, we tried to convert to the method of enzyme digestion and ligation to solve this problem.</p>  
                    <h1 class="sectitle title-cover" id="sectitle3">CYCLE 2: constructed by double enzyme digestion and enzyme ligation method</h1>
                    <h2 class="subtitle" id="list5">   
                        2.1 Design
                    </h2> 
                        <p>After many failed attempts, we decided to try double enzyme digestion and enzyme ligation to construct our plasmids.</p>
                        
                        <p>We first perform a double enzyme digestion reaction to obtain multiple fragments and vectors with the same sticky ends, and then ligate the carriers and fragments containing the same sticky ends together through an enzyme ligation reaction. We hope it will solve the problem of low success rate of homologous recombination in dealing with multi-fragment and large-fragment at this stage.</p>
                        <p>In the first step, we use the double enzyme digestion method to obtain the target gene fragments and the vector fragment with sticky ends. We first culture the basal plasmid-containing strain and extract the plasmid, then add the digestion site to the  target gene fragment and the vector fragment by PCR, and then obtain the target gene fragment and the vector fragment with the same sticky end by double enzyme digestion. For dsup-gshA-gshB fragments of <i>Synechocystis</i> PCC6803 and ycel-pncB-nadE- nadD-nadM fragments of <i>S.oneidensis</i> MR-1, we directly cleavage the extracted basal plasmid by double enzyme digestion to obtain gene fragments with sticky ends; For the ldhA-lldP-omcs fragment of <i>Synechocystis</i> PCC6803 and the dsup-sodA-oprf fragment of <i>S.oneidensis</i> MR-1, we first PCR the extracted plasmid to obtain the target gene fragments, and then obtain the fragment with sticky end by double enzyme digestio</p>
                        <p>In the second step, we use enzyme ligation to link the target gene fragment and its corresponding vector fragment with the same sticky end together, and finally construct a complete plasmid.</p>
                        <img src="https://static.igem.wiki/teams/4595/wiki/engineering/engineering/5.png" alt="" class="normimg1" >
                        <i class="imgexplain">Fig 4. The diagram of constructing the plasmid inserted with pLactate by double enzyme digestion and enzyme ligation</i>
                        <img src="https://static.igem.wiki/teams/4595/wiki/engineering/engineering/6.png" alt="" class="normimg1" >
                        <i class="imgexplain">Fig 5. The diagram of constructing the plasmid inserted with pResistance6803 by double enzyme digestion and enzyme ligation</i>
                        <img src="https://static.igem.wiki/teams/4595/wiki/engineering/engineering/7.png" alt="" class="normimg1" >
                        <i class="imgexplain">Fig 6. The diagram of constructing the plasmid inserted with pResistance MR-1 by double enzyme digestion and enzyme ligation</i>
                        <img src="https://static.igem.wiki/teams/4595/wiki/engineering/engineering/8.png" alt="" class="normimg1" >
                        <i class="imgexplain">Fig 7. The diagram of constructing the plasmid inserted with pElectricity by double enzyme digestion and enzyme ligation</i>
                    <h2 class="subtitle" id="list6">   
                        2.2 Build
                    </h2>
                        <p>Our basic process of constructing a complete plasmid is: extract the plasmid, perform PCR to obtain the fragment containing the target gene, obtain the fragment of the target gene with sticky ends through double enzyme digestion, and use the enzyme ligation reaction to link the gene of interest to the vector.</p>
                        <p>Take the pLactate system pathway as an example: we first obtain linearized vector fragments containing ampicure resistance by PCR, and then obtain gene fragments and vector fragments by gel extraction, then verify and purify gene fragments and vector fragments. We then double-enzyme-digested the fragments with restriction enzymes SpeI and EcorI to obtain gene and vector fragments with identical sticky ends. Finally, we used an enzyme ligation reaction to ligate a fragment containing the gene ldhA-lldP-omcs with the vector fragment corresponding to it to construct a complete plasmid. We then transferred the constructed plasmid into <i>E. coli</i> TOP10 and seeded into LB plates containing 100 μg/mL ampicin for screening. When single colonies appeared on the surface of solid media, we singled out the single colonies for PCR validation. After observing the correct bands, we mixed 800 μl of bacterial solution with 800 μl of 1:1 mixture of glycerol and double distilled water and keep the seeds at -80℃.</p>
                    <h2 class="subtitle" id="list7">
                        2.3 Result
                    </h2>
                        <p>After several attempts, we successfully constructed plasmids pLactate, pResistance6803, pElectricity, pResistance MR-1.</p>
                       <img src="https://static.igem.wiki/teams/4595/wiki/engineering/engineering/mqml1.png" alt="" class="normimg1">
                        <i class="imgexplain">Fig 8. The left figure is the result of PCR identification of pLactate plasmid.It shows that there is a bright band at about 5000 bp, and the length of pLactate plasmid is 5479 bp, which proves that we have successfully constructed the plasmid of pLactate system pathway.</i>
                        <i class="imgexplain">The right figure is the result of PCR identification of pResistance6803 plasmid.It shows that there is a bright band at about 5000 bp, and the length of pResistance6803 plasmid is 4746 bp, which proves that we have successfully constructed the plasmid of pResistance6803 system pathway.</i>
                        <img src="https://static.igem.wiki/teams/4595/wiki/engineering/engineering/mqml2.png" alt="" class="normimg1" >
                        <i class="imgexplain">Fig 9. The left figure is the result of PCR identification of pElectricity plasmid.It shows that there is a bright band at about 5000 bp, and the length of pElectricity plasmid is 5736 bp, which proves that we have successfully constructed the plasmid of pElectricity system pathway.</i>
                        <i class="imgexplain">The right figure is the result of PCR identification of pResistance MR-1 plasmid.It shows that there is a bright band at about 3000 bp, and the length of pResistance MR-1 plasmid is 3291 bp, which proves that we have successfully constructed the plasmid of pResistance MR-1 system pathway.</i>
                        <p></p>
                    <h2 class="subtitle" id="list8">
                        2.4 Experience
                    </h2>
                        <p>At first, we encountered the problem that the digestion reaction could not be carried out due to the low concentration of the recovered PCR fragments. After several attempts, we found that the extraction of PCR fragment products, the timing and temperature of the double enzyme digestion reaction and the enzyme ligation reaction had a great influence on the success rate of plasmid construction. Following numerous setbacks, we meticulously analyzed and documented the underlying causes. After conducting a series of experiments, we successfully extracted the necessary fragments and vectors from fluid samples, maximizing their concentrations for double enzyme digestion. After double enzyme digestion, we obtained the double enzyme digestion product by gel extraction to ensure the purity of the fragment. At the same time, we also continuously adjusted the reaction temperature and narrowed The time range of the reaction, and finally found the most suitable reaction time and temperature system. After a series of difficulties, we figured out an optimally constructed system and obtained the corresponding complete plasmid.</p>
                        <p>Electrico sisters is of great significance to solve the energy crisis and protect the global environment, encouraging us to use synthetic biology to make some contributions in this regard. We will continue to carry out this topic and are willing to do our part for the development of synthetic biology.</p>
                    <h1 class="sectitle title-cover" id="sectitle4">CYCLE 3: Expression validation</h1> 
                    <h2 class="subtitle" id="list9">
                        3.1 <i>Synechocystis</i> PCC6803
                    </h2>
                        <h3 class="subsubtitle">3.1.1 Design</h3>
                        <p>We hope to figure out whether the growth cycle of <i>Synechocystis</i> PCC6803 engineered bacteria is shorter than that of wild-type Synechocyst PCC6803. Therefore, we detect the growth of <i>Synechocystis</i> PCC6803 and plot its growth curve. At the same time, we verify whether the efficiency of the strain we constructed achieves our expectations by detecting the concentration of expression products.</p>
                        <h3 class="subsubtitle">3.1.2 Build</h3>
                        <p>First, we culture the <i>E. coli</i> TOP10 strain containing the correct plasmid and extracted the plasmid. Next, we transfer the extract plasmid into <i>Synechocystis</i> PCC6803. The experimental operation is: we use BG11 liquid medium to resuspend and wash wild-type <i>Synechocystis</i> PCC6803 twice, take an appropriate amount of <i>Synechocystis</i> PCC6803 solution and add an appropriate amount of plasmid, stand and activate it for a period of time, apply the bacterial solution to BG11 plate with corresponding resistant. When the transformant appears on the surface of the solid medium, we pick out the transformant for PCR verification. After observing the correct bands, we mix 800 μl of <i>Synechocystis</i> PCC6803 solution with 800 μl of 1:1 mixture of glycerol and double distilled water and keep the seeds at -80℃.</p>
                        <h3 class="subsubtitle">3.1.3 Test</h3>
                        <p>We examined the growth curves of wild-type <i>Synechocystis</i> PCC6803 and <i>Synechocystis</i> PCC6803 engineered bacteria omcs. We compared the growth curves and found that the growth cycle of <i>Synechocystis</i> PCC6803 engineered bacteria omcs was shorter, which proved that the genes we transferred into <i>Synechocystis</i> PCC6803 were finally effectively expressed.
</p>
                        <p>At the same time, we detected the amount of lactate produced by wild-type <i>Synechocystis</i> PCC6803 and <i>Synechocystis</i> PCC6803 engineering bacteria pLactate. Comparing the data of wild-type and engineered bacteria pLactate, we found that the amount of expression products of <i>Synechocystis</i> PCC6803 engineering bacteria pLactate were higher than those of wild-type <i>Synechocystis</i> PCC6803, demonstrating that the strain we have built works as expected.
</p>
                    <h2 class="subtitle" id="list10">
                        3.2 <i>Shewanella oneidensis </i> MR-1
                    </h2>
                        <h3 class="subsubtitle">3.2.1 Design</h3>
                        <p>We hope to verify that the strain we built worked as efficiently as expected by measuring the amount of the expression product NADH/NAD+. At the same time, we compared whether the ability to generate electricity of <i>S.oneidensis</i> MR-1 transferred with the plasmid we constructed was stronger than that of wild-type <i>S.oneidensis</i> MR-1 by measuring the efficiency of electricity production from Shewanella.</p>
                        <h3 class="subsubtitle">3.2.2 Build</h3>
                        <p>We culture the strain containing the correct plasmid and extract the plasmid to transfer into <i>E. coli</i> WM3064. Subsequently, we transferred the plasmid we constructed into <i>S.oneidensis</i> MR-1 by conjugation between <i>E. coli</i> WM3064 and <i>S.oneidensis</i> MR-1. The experimental operation is: we inoculated an equal and appropriate amount of <i>E. coli</i> WM3064 with <i>S.oneidensis</i> MR-1 into 1 ml liquid LB medium containing 0.019 g/mL DAP and 50 μg/mL kanamycin for one hour, and finally inoculated the mixed bacteria solution on LB plates containing 50 μg/mL / mL of kanamycin. When single colonies of <i>S.oneidensis</i> MR-1 appeared on the surface of solid media, we singled out the single colonies for PCR validation. After observing the correct bands, we mixed 800 μl of <i>S.oneidensis</i> MR-1 solution with 800 μl of 1:1 mixture of glycerol and double distilled water and keep the seeds at -80℃.</p>
                        <h3 class="subsubtitle">3.2.3 Test</h3>
                        <p>We measured the amount of NADH/NAD+ produced by wild-type <i>S.oneidensis</i> MR-1 and <i>S.oneidensis</i> MR-1 engineered bacteria ycel-pncB and nadE-nadD-nadM. Comparing the amount of NADH/NAD+ produced by wild-type <i>S.oneidensis</i> MR-1 with the amount of NADH/NAD+ produced by <i>S.oneidensis</i> MR-1 engineering bacteria ycel-pncB and nadE-nadD-nadM, we found that the expression products of ycel-pncB and nadE-nadD-nadM were higher than those of wild-type <i>S.oneidensis</i> MR-1, which proved that the working efficiency of the strain we constructed achieved our expectations.</p>
                        <p>At the same time, we tested the electricity production of wild-type <i>S.oneidensis</i> MR-1 and <i>S.oneidensis</i> MR-1 engineering bacteria ycel-pncB and nadE-nadD-nadM. Comparing the electricity production of the two, we found that the voltage of the battery constructed by <i>S.oneidensis</i> MR-1 engineering bacteria ycel-pncB and nadE-nadD-nadM was higher than that of the battery constructed by wild-type <i>S.oneidensis</i> MR-1.</p>
                </article>
            </div>
        </div>
    </div>
</div>
    <div class="footerdivide"></div>
<a href="javascript:scrollTo(0, 1000000)"><img src="https://2022.igem.wiki/hust-china/static/arrow-down-square.svg" alt="arrow-down-square"
      id="to-the-bottom"></a>
  <a href="javascript:scrollTo(0, 0)"><img src="https://2022.igem.wiki/hust-china/static/arrow-up-square.svg" alt="arrow-up-square" id="to-the-top"></a>
<script>
    var herfsvg = document.getElementById("herfsvg")
    herfsvg.style.left = "48%";
    herfsvg.style.top = "90%";
    var titlebg = document.getElementById("titlebg")
    titlebg.style.height = (window.screen.height - 100)   + "px"//"200px"
    window.onscroll = function () {
      var currentHeight =
        document.documentElement.scrollTop ||
        window.pageYOffset ||
        document.body.scrollTop;
      // 页面滚动超过300px就显示
      if (currentHeight > 300) {
        document.getElementById('to-the-top').style.display = 'block'
      } else {
        document.getElementById('to-the-top').style.display = 'none'
      }
      if (currentHeight < 900000) {
        document.getElementById('to-the-bottom').style.display = 'block'
      } else {
        document.getElementById('to-the-bottom').style.display = 'none'
      }
    }
  </script>


    <!--  Video  -->
  </div>

  <!-- Footer: MUST mention license AND have a link to team wiki's repository on gitlab.igem.org -->
  <footer class="footer">
</footer>

  <!-- Wiki Tools: Teams are allowed to remove it -->
  <style>
    .wiki-tools-footer {
        --bs-gutter-x: 1.5rem;
        background: #222124;
        margin: 0;
        max-width: 100%;
        padding: 1rem 3rem;
        display: flex;
        gap: 1.5em;
    }
    .wiki-tools-footer a {
        text-decoration: none;
        color: white;
        font-weight: 400;
        transition: color 0.3s;
    }
    .wiki-tools-footer a:hover {
        color: #08C784;
    }
    .wiki-tools-footer img {
        object-fit: contain;
    }
</style>

<section class="wiki-tools-footer container">
    <a id="wiki-tools-link">Access Wiki Tools</a>
    <a id="wiki-tools-repo">View Source Code</a>
    <img id="wiki-tools-pipeline">
</section>

<script>
    const wikiToolsLink = document.getElementById('wiki-tools-link')
    const wikiToolsRepo = document.getElementById('wiki-tools-repo')
    const wikiToolsPipeline = document.getElementById('wiki-tools-pipeline')

    const path = document.location.pathname.split('/')

    const competitionSlug = document.location.host.split('.')[0]
    const teamSlug = path[1]
    const page = path[2] === '' ? 'index' : path[2]

    wikiToolsLink.href = `https://tools.igem.org/wiki-tools/${competitionSlug}/${teamSlug}/${page}`
    wikiToolsRepo.href = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}`
    wikiToolsPipeline.src = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}/badges/main/pipeline.svg`
</script>

  <!-- Bootstrap Bundle with Popper -->
  <script src="static/bootstrap.bundle.min.js"></script>
  </body>
  
</html>
