<head>
  <meta charset="utf-8">
  <title>Results | HUST-China-igem 2023</title>
  <meta content="width=device-width, initial-scale=1" name="viewport">
  <meta content="Webflow" name="generator">
  <link href="static/normalize.css" rel="stylesheet" type="text/css">
  <link href="static/webflow.css" rel="stylesheet" type="text/css">
  <link href="static/igem-hust-china-2023.other_pages.css" rel="stylesheet" type="text/css">
  <link href="https://fonts.googleapis.com" rel="preconnect">
  <link rel="license" href="https://creativecommons.org/licenses/by/4.0/" />
  <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin="anonymous">
  <script src="static/jquery.js" type="text/javascript"></script>
  <script src="static/webfont.js" type="text/javascript"></script>
  <script src="static/webflow.js" type="text/javascript"></script>
  <script type="text/javascript">
    WebFont.load({
      google: {
        families: ["Inconsolata:400,700", "Montserrat:100,100italic,200,200italic,300,300italic,400,400italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic", "Droid Serif:400,400italic,700,700italic", "Oswald:200,300,400,500,600,700", "Merriweather:300,300italic,400,400italic,700,700italic,900,900italic"]
      }
    });
  </script>
  <script type="text/javascript">
    ! function (o, c) {
      var n = c.documentElement,
        t = " w-mod-";
      n.className += t + "js", ("ontouchstart" in o || o.DocumentTouch && c instanceof DocumentTouch) && (n.className += t + "touch")
    }(window, document);
  </script>
  <link href="images/favicon.ico" rel="shortcut icon" type="image/x-icon">
  <link href="images/webclip.png" rel="apple-touch-icon">


<style>
    #subtitle1:target {
        padding-top: 60px;
        margin-top: -60px;
    }


    #to-the-top {
        /*svg导航*/
        height: 30px;
        width: 30px;
        bottom: 150px;
        right: 0px;
        position: fixed;
        display: none;
        /*padding: 10px;*/
        margin: 10px;
    }

    #to-the-bottom {
        height: 30px;
        width: 30px;
        bottom: 110px;
        right: 0px;
        position: fixed;
        /*padding: 10px;*/
        margin: 10px;
    }
    body{
      margin=0;
    }
</style>

<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="https://static.igem.wiki/common/icons/favicons/igem-2022.svg"/>
    <link rel="license" href="https://creativecommons.org/licenses/by/4.0/"/>

    <!-- Bootstrap CSS -->
    <link href="static/bootstrap.min.css" rel="stylesheet">

    <!-- JQuery Js -->
    <script type="text/javascript" src="static/jquery.js")}}" ></script>
    <script type="text/javascript" src="static/jqueryrotate.js")}}" ></script>


    <!-- Custom CSS -->
    <link href="static/style.css" rel="stylesheet">

    <!-- Font style-->
    
    <title> | HUST-China - iGEM 2022</title>
    <style>
      @font-face {
	font-family:Silkscreen;
	src: url(static/Silkscreen-Regular.ttf);
}
@font-face {
    font-family: Montserrat;
    src: url(static/Montserrat-Regular.ttf);
}

@font-face {
    font-family: Mukta;
    src: url(static/Mukta-Regular.ttf);
}

@font-face {
    font-family: Ubuntu;
    src: url(static/Ubuntu-Regular.ttf);
}

@font-face {
    font-family: Josefin_Sans;
    src: url(static/JosefinSans-Regular.ttf);
}

      </style>
    
  </head>
<body class="body">
<div data-animation="default" class="navbar-logo-left-container shadow-three w-nav" data-easing2="ease-out"
    data-easing="ease-in" data-collapse="all" role="banner" data-no-scroll="1" data-duration="400" id="page-header"
    data-doc-height="1">
    <div class="navbar-wrapper">
      <div class="w-layout-blockcontainer container-2 w-container">
        <img src="https://static.igem.wiki/teams/4595/wiki/home/logo.png" loading="lazy" alt="" class="team-logo"></div>
      <nav role="navigation" class="nav-menu-wrapper w-nav-menu">
        <ul role="list" class="nav-menu-two w-list-unstyled">
          <li>
            <a href="index.html" target="_self" aria-current="page" class="nav-link w--current">Home</a>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1850" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1851" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div class="text-block-7"><span>Project</span></div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="description"
                  class="nav-dropdown-link w-dropdown-link">Description</a>
                <a href="safety" class="nav-dropdown-link w-dropdown-link">Safety</a>
                 <a href="implementation"
                  class="nav-dropdown-link w-dropdown-link">Proposed<br>Implementation</a>

              </nav>
            </div>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="f73ff7f3-a811-76cc-69d5-a6519c08fec2" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="f73ff7f3-a811-76cc-69d5-a6519c08fec3" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div class="text-block-7">Wet Lab</div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="design" class="nav-dropdown-link w-dropdown-link">Design</a>
                <a href="engineering"
                  class="nav-dropdown-link w-dropdown-link">Engineering</a>
                <a href="protocol" class="nav-dropdown-link w-dropdown-link">Protocol</a>
                <a href="results" class="nav-dropdown-link w-dropdown-link">Results</a>
                <a href="notebook" class="nav-dropdown-link w-dropdown-link">Notebook</a>
                <a href="parts" class="nav-dropdown-link w-dropdown-link">Parts</a>
              </nav>
            </div>
          </li>
          <li>
            <a href="hardware" class="nav-link">Hardware</a>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1869" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f186a" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div>HP</div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="human-practices"
                  class="nav-dropdown-link w-dropdown-link">Human<br>Practices</a>
                <a href="education"
                  class="nav-dropdown-link w-dropdown-link">Education&amp;<br>Communication</a>
              </nav>
            </div>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1845" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1846" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div>Team</div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="contribution"
                  class="nav-dropdown-link w-dropdown-link">Contribution</a>
                <a href="members" class="nav-dropdown-link w-dropdown-link">Members</a>
                <a href="attributions"
                  class="nav-dropdown-link w-dropdown-link">Attributions</a>
              </nav>
            </div>
          </li>
        </ul>
      </nav>
      <div class="menu-button w-nav-button">
        <div class="icon w-icon-nav-menu"></div>
      </div>
    </div>
</div>

  
  <!-- Header -->
    
<header>

<a href="#sectitle1"><img src="https://2022.igem.wiki/hust-china/static/arrow-bar-down.svg" alt="" id="herfsvg"></a>
<a href="#sectitle1">
  <div id="titlebg" style="background: url(https://static.igem.wiki/teams/4595/wiki/results/results/results.jpg); background-size:cover"></div>
  </a>
</header>

  <!-- Page Content -->
  <div class="container">
    
 <div class="container">
        <div class="catalogue">
            <h4>content</h4>
            <ul>
                <li><a href="#sectitle1">Introduction</a></li>
                <li><a href="#sectitle2">Plasmid amplification and reconstruction </a></li>
                <ul>
                  <li><a href="#list1"> Plasmids for Cyanobacteria </a></li>
                  <li><a href="#list2"> Plasmids for Shewanella</a> </li>
                </ul>
                <li><a href="#sectitle3">Strain construction</a></li>
                
                <li><a href="#sectitle4">Expression Validation</a></li>
                <ul>
                  <li><a href="#list3"> Cystidia PCC6803</a></li>
                  <li><a href="#list4"> Shewanella MR-1</a></li>
                  <li><a href="#list5"> Co-culture system</a></li>
                </ul>
            </ul>
        </div>
    <div class="main content">
        <div class="row">
            <div class="col-lg-1"></div>
            <div class="content col-lg-10">
                <article>
                  <h1 class="sectitle title-cover" id="sectitle1">Section 1 : Introduction</h1>
                        <p>Our project aimed to enable Cyanobacteria to produce lactic acid at a high efficiency, enhance the electricity producing capacity of Shewanella, as well as improve the resistance of both bacteria. In order to achieve these goals, we construct mainly 4 plasmids: pLactate and pResistence6803 for Cyanobacteria, pElectricity and pResistance MR-1 for Shewanella. Besides 4 main plasmids, we also constructed 3 plasmids in order to test the function of genes more conveniently. </p>
                         <p>We transferred them into Synechocystis PCC6803 and Shewallena MR-1 for expressing the proteins we needed. At the same time, in order to verify the efficiency of electricity production, we also carried out and verified our project from all aspects.</p>
                    <h1 class="sectitle title-cover" id="sectitle2">Section 2 : Plasmid amplification and reconstruction </h1>
                     <p>We first transformed the plasmids containing designed sequence synthesized by the company into  Escherichia coli Top10 for amplification, so we have enough amount of plasmid to carry out subsequent experiments. Then colony PCR showed a successful transformation, and required plasmids were extracted for the next experiment. </p>
                      <p>We then obtained target fragments by means of PCR, and then constructed them into the plasmids we need. </p>
                    <h2 class="subtitle" id="list1">Plasmids for Cyanobacteria </h2>
                        <p>We have 2 plasmids, pLactate and pResistance6803 for Cyanobacteria. We verified the size of each plasmids as well as all the fragments involved in constructing the plasmids. In addition to pLactated and pResistance, we also constructed plasmid Omcs to ensure the function of it individually.</p>
                        <p>pLactate includes LdhA enabling Cyanobacteria to produce lactic acid by means of photosynthesis, Lldp to transform intracellular lactate to the extracellular as well as Omcs improving the efficiency of electricity transformation.</p>
                         <img src="https://static.igem.wiki/teams/4595/wiki/safety/safety4-1.png" alt="" class="normimg" >
                        <img src="https://static.igem.wiki/teams/4595/wiki/results/results/1-gh.png" alt="" class="normimg" >
                        <i class="imgexplain">Fig. 1. The PCR result of Pcpc560-ldhA-lldP-Tpsbc-Ppsba2-omcs-Tpsbc.</i>
                        <p>The bands of Pcpc560-ldhA-lldP-Tpsbc-Ppsba2-omcs-Tpsbc(5000+bp) from PCR are identical to the theoretical lengths of 5479bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained.</p>

                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/2-g-1.png" alt="" class="normimg">
                       <i class="imgexplain">Fig. 2. The PCR result of Pcpc560-ldhA-lldP-Tpsbc.</i>
                       <p>The bands of Pcpc560-ldhA-lldP-Tpsbc(3000+bp) from PCR are identical to the theoretical lengths of 3643bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained.</p>
                     <img src="https://static.igem.wiki/teams/4595/wiki/results/results/3-1-g-2-g-2.png" alt="" class="normimg">
                     <img src="https://static.igem.wiki/teams/4595/wiki/results/results/3-2-g-2-gh.png" alt="" class="normimg">
                       <i class="imgexplain">Fig. 3. The PCR result of Ppsba2-omcs-Tpsbc from pLactate(left) and Omcs(right). </i>
                     <p>The bands of Ppsba2-omcs-Tpsbc(almost 2000bp) from PCR are identical to the theoretical lengths of 1866bp(from pLactate) and 1820bp(from Omcs) estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained.</p>
                      <img src="https://static.igem.wiki/teams/4595/wiki/results/results/4-0168.png" alt="" class="normimg">
                       <i class="imgexplain">Fig. 4. The PCR result of PUC57-slr0168(vector).</i>
                        <p>The bands of PUC57-slr0168(vector)(almost 5000bp) from PCR are identical to the theoretical lengths of 4727bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained.</p>
                         <p>All above results showed us a success of the construction of pLactate and Omcs.</p>
                         <p>pResistance6803 is constructed to enhance the resistance of Cyanobacteria under adversities, especially the antioxidant capacity and the ability to protect DNA from damages. </p>
                         <img src="https://static.igem.wiki/teams/4595/wiki/safety/safety3-1.png" alt="" class="normimg">
                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/5-lir.png" alt="" class="normimg">
                       <i class="imgexplain">Fig. 5. The PCR result of fragment PrbcL-dsup-Tpsbc-PrbcL-gshA-gshB-Tpsbc.</i>
                       <p>The bands of PrbcL-dsup-Tpsbc-PrbcL-gshA-gshB-Tpsbc(almost 5000bp) from PCR are identical to the theoretical lengths of 4746 estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained. </p>
                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/6-l-1.png" alt="" class="normimg">
                       <i class="imgexplain">Fig. 6. The PCR result of fragment PrbcL-dsup-Tpsbc.</i>
                       <p>The bands of PrbcL-dsup-Tpsbc	(almost 2000bp) from PCR are identical to the theoretical lengths of 1848 estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained. </p>
                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/7-l-2.png" alt="" class="normimg">
                       <i class="imgexplain">Fig. 7. The PCR result of fragment PrbcL-gshA-gshB-Tpsbc.</i>
                       <p>The bands of PrbcL-gshA-gshB-Tpsbc(almost 3000bp) from PCR are identical to the theoretical lengths of 2889 estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained. </p>
                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/8-2030.png" alt="" class="normimg">
                       <i class="imgexplain">Fig. 8. The PCR result of fragment PUC57-slr2030.</i>
                       <p>The bands of PUC57-slr2030(almost 5000bp) from PCR are identical to the theoretical lengths of 4633 estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained. </p>
                       <p>All above results showed us a success of the construction of pResistance6803. </p>
                    <h2 class="subtitle" id="list2">   
                        Plasmids for Shewanella
                    </h2> 
                        <p>We have 2 plasmids, pElectrcity and pResistanceMR-1 for Shewallena. We verified the size of each plasmids as well as all the fragments involved in constructing the plasmids . Besides pElectricity and pResistanceMR-1, we also constructed plasmids to ensure the function of ycel-pncB and nadE-nadD-nadM individually.</p>
                        <p>pElectricty is constructed with ycel-pncB and nadE-nadD-nadM, both can elevate the electricity production rate of Shewanella. </p>
                        <img src="https://static.igem.wiki/teams/4595/wiki/safety/safety3-1.png" alt="" class="normimg">
                        <img src="https://static.igem.wiki/teams/4595/wiki/results/results/9-cd.png" alt="" class="normimg">
                         <i class="imgexplain">Fig.9. The PCR result of fragment Ptac-ycel-pncB-TrrnB T1-Ptac-nadE-nadD-nadM-TrrnB T1.</i>
                        <p>The bands of Ptac-ycel-pncB-TrrnB T1-Ptac-nadE-nadD-nadM-TrrnB T1(5000+bp) from PCR are identical to the theoretical lengths of 5736 estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained.</p>
                         <img src="https://static.igem.wiki/teams/4595/wiki/results/results/10-c-1-c-2-cd.png" alt="" class="normimg">
                         <i class="imgexplain">Fig.10. The PCR results of Ptac-ycel-pncB-TrrnB T1 and Ptac-nadE-nadD-nadM-TrrnB T1 from pElectricity.</i>
                          <p>The bands of Ptac-ycel-pncB-TrrnB T1(2500+bp) and Ptac-nadE-nadD-nadM-TrrnB T1(about3000 bp) from PCR are identical to the theoretical lengths of 2601bp and 2820bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained.</p>
                          <img src="https://static.igem.wiki/teams/4595/wiki/results/results/11-c-1.png" alt="" class="normimg">
                         <i class="imgexplain">Fig.11. The PCR results of fragment Ycel-PncB from ycel-pncB.</i>
                          <p>TThe bands of Ycel-PncB(about 2500bp) from PCR are identical to the theoretical lengths of 2622bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained.</p>
                          <img src="https://static.igem.wiki/teams/4595/wiki/results/results/12-c-2.png" alt="" class="normimg">
                         <i class="imgexplain">Fig. 12. The PCR results of Ptac-nadE-nadD-nadM-TrrnB T1 from nadE-nadD-nadM.</i>
                          <p>The bands of Ptac-nadE-nadD-nadM-TrrnB T1 (about 3000bp) from PCR are identical to the theoretical lengths of 2820bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained.</p>
                           <img src="https://static.igem.wiki/teams/4595/wiki/results/results/13-pyydt.png" alt="" class="normimg">
                         <i class="imgexplain">Fig.13. The PCR results of PYYDT(vector).</i>
                          <p>The bands of PYYDT(vector)	(5000+bp) from PCR are identical to the theoretical lengths of 5886bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained.</p>
                          <p>All above results showed us a success of the construction of pElecteicity, ycel-pncB and nadE-nadD-nadM.</p>
                          <p>pResistance MR-1 is constructed with dsup-sodA and oprf, the former can lower the level of ROS in Shewallena and avoid DNA damages, and the latter can assist in the entry and exit of riboflavin and improves the efficiency of electron transfer  </p>
                        <img src="https://static.igem.wiki/teams/4595/wiki/safety/safety3-1.png" alt="" class="normimg">
                        <img src="https://static.igem.wiki/teams/4595/wiki/results/results/14-sir.png" alt="" class="normimg">
                         <i class="imgexplain">Fig.14. The PCR result of Ptac-dsup-sodA-oprf-TrrnB T1.</i>
                         <p>The bands of Ptac-dsup-sodA-oprf-TrrnB T1(3000+bp) from PCR are identical to the theoretical lengths of 3291bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these fragments had successfully been obtained.</p>
                           <img src="https://static.igem.wiki/teams/4595/wiki/results/results/15-s-12.png" alt="" class="normimg">
                         <i class="imgexplain">Fig.15. The PCR result of fragment Ptac-dsup-sodA-TrrnB T1.</i>
                         <p>The bands of Ptac-dsup-sodA-oprf-TrrnB T1(3000+bp) from PCR are identical to the theoretical lengths of 3291bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these fragments had successfully been obtained.</p>
                           <img src="https://static.igem.wiki/teams/4595/wiki/results/results/16-s-3.png" alt="" class="normimg">
                         <i class="imgexplain">Fig.16. The PCR result of fragment Ptac-oprf-TrrnB T1.</i>
                          <p>The bands of Ptac-oprf-TrrnB T1(1000+bp) from PCR are identical to the theoretical lengths of 1135bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these fragments had successfully been obtained.</p>
                          <p>The results above could demonstrate that these plasmids are correctly constructed, so we can transform pLactate, pResistence6803,and omcs into Synechocystis PCC6803, while pElectricity, ycel-pncB, nadD-nadE-nadM and pResistance MR-1into Shewallena MR-1 strains for verification.   </p>
                           
                     <h1 class="sectitle title-cover" id="sectitle3">Section 3 :Strain construction</h1>
                      <p>We have successfully constructed all the target plasmids needed in the four loops, and this step really facilitates our subsequent experiments. At the same time, in order to facilitate the measurement of the function of some elements, we separately constructed plasmids (omcs, ycel-pncB, Nad-Nadd-nadM) for some essential elements.</p>
                      <img src="https://static.igem.wiki/teams/4595/wiki/results/results/results1.png" alt="" class="normimg1">
                         <i class="imgexplain">Fig.17. Colony PCR results of pLactate, Omcs and pResistence6803.</i>
                         <img src="https://static.igem.wiki/teams/4595/wiki/results/results/results2.png" alt="" class="normimg1">
                         <i class="imgexplain">Fig.18. Colony PCR results of target fragments in plasmids pElectricity, ycel-pncB, nadD-nadE-nadM and pResistenceMR-1.</i>
                       <p>All the bands are identical to the theoretical sizes, demonstrating that these plasimds had successfully been transformed and verified.The results are as follows: PLactate: Pcpc560 - ldhA - lldP - Tpsbc - Ppsba2 - omcs -Tpsbc about 5400 bp , pResistance6803: PrbcL - dsup - Tpsbc - PrbcL - gshA - gshB -Tpsbc about 4100 bp , omcs: Ppsba2-omcs-Tpsbc about 1800bp, pElectricty:Ptac-ycel-pncB-TrrnB T1-Ptac-nadD-nadE-nadM-TrrNB T1 about 5700bp, pResistance MR-1: Ptac-dsup-soda-oprf-TrrnB T1 about 3200bp , ycel-pncB: Ptac-Ycel-PNCB-TrrnB T1 about 2500bp, nadD-nadE-nadM: Ptac-nadD-nadE-nadM-TrrNB T1 about 2800bp</p>
                       <p>The theoretical length of 5461bp, 4107bp,1809bp, 5666bp, 3229bp,  2564bp and 2820bp estimated by the design primers location (promoter to terminator) was the same, indicating that we successfully obtained the plasmid we needed.</p>
                       <p>After the successful plasmid transformation into <i>Synechocystis</i> PCC6803 and <i>S.oneidensis MR-1</i> was verified by colony PCR.</p>
                       <h1 class="sectitle title-cover" id="sectitle4">Section 4 : Expression Validation </h1>
                       <h2 class="subtitle" id="list3">   
                      <i>Synechocystis</i> PCC6803
                    </h2> 
                     <p>4.1.1 Lactate production verification</p>
                       <p>We co-culture <i>Synechocystis</i> PCC6803 and <i>S.oneidensis MR-1</i>. The nutrient of <i>S.oneidensis MR-1</i> is lactate produced by <i>Synechocystis</i> PCC6803. We tried to determine the production of lactate by HPLC to test the feasibility of the co-culture method. We first measured the standard curve of lactate concentration and determined its retention time. The <i>Synechocystis</i> PCC6803(pLactate) cells were broken by freeze-thaw method, and the supernatant after centrifugation was taken for high performance liquid chromatography (HPLC). A peak with almost the same retention time which is 3.912 min as the lactate standard substance was observed on the spectrum, which confirmed the successful expression of lactic acid in engineered bacteria. The peak area was compared with the standard curve. It can be seen that the lactic acid yield of our engineering bacteria cyanobacteria was (see figure), indicating that the pLactate gene contained in the plactate gene could produce lactic acid stably and provide nutrition for Shivaella stably. This result indicated the successful expression of the imported LdhA and LldP genes, and verified the feasibility of the co-culture system.</p>
                       <p>4.1.2 Growth of cystis aggregatus</p>
                       <p>We used BG11 to culture the engineered bacteria and wild type containing omcs respectively, and measured the OD value of the culture medium at 730nm every 24 hours. A growth curve is obtained as follows. The results showed that the growth rate of the engineered bacteria was faster than that of the wild type, which achieved the expected effect of omcs transfer, and proved that omcs was successfully expressed in the anthocystosis.</p>
                       <h2 class="subtitle" id="list4">   
                        Shewanella MR-1
                    </h2> 
                     <p>4.2.1 Determination of NADH/NAD+ concentration</p>
                       <p>To verify normal gene expression in Shewanella, we measured the NADH/NAD+ content of wild type and engineered strains ycel-pncB and NAD-Nadd-NADM, respectively. The NADH/NAD+ content in wild-type MR-1 and engineered bacteria was compared as follows. It was found that the NADH/NAD+ content in Shewanella containing transgenic ycel-pncBC1 and NAD-Nadd-NADM was significantly higher than that in wild-type Shewanella, which verified that we successfully introduced the target gene. And it is normally expressed in Shewanella.</p>
                       <p>4.2.2 Verification of electrification of Shewanella</p>
                       <p>We built a tiny cell (pictured) to test the electricity-producing performance of a log-growing Shewanella.</p>
                       <p>Cell construction: We inoculated Shivanella cultured overnight in 100 mL LB medium and cultured it at 30℃ in a 200 rpm shaker to OD600 to 0.6-0.8. The cultured cells were collected by centrifuge and re-suspended in 140 mL anoditic solution, and 50 μg/mL kana antibiotics were added to maintain constant culture conditions and 0.1 mM IPTG inducer to initiate gene expression. The two electrode chambers of MFC are separated by proton exchange membrane, using carbon cloth as cathode and anode electrodes, and then adding 140mL cathode solution to the cathode chamber, using 2kΩ external resistance to close the external circuit. The specific battery structure of a complete MFC is as follows:</p>
                        <p>1) testing the micro battery</p>
                       <p>Using the micro battery, we use a digital multimeter to measure the output voltage of microbial fuel cell. We measured the voltage of the ycel-pncB, NAD-Nadd-NADM gene pathway in Shiwanella and compared it with the voltage of wild-type Shiwanella. Measure a set of voltages every three hours and compare as follows. The results showed that the voltage generated by the gene pathway containing ycel-pncB and NAD-Nadd-NADM in the logarithmic growth stage of Shiwanella was significantly higher than that of wild-type Shiwanella, which verified that we successfully introduced the target gene and its normal expression and function in Shiwanella.</p>
                       <p>2) Photosensitizer & riboflavin</p>
                       <p>In our search for relevant papers, we learned that photosensitivities and riboflavin contribute to extracellular electron transfer, thereby improving the efficiency of electricity generation. Therefore, we investigated the effect of addition of photosensitizers (eosin Y, fluorescein) and riboflavin on electricity production in Shiwanella. We compared the electrical production of the experimental group supplemented with photosensitizers (eosin Y, fluorescein) and riboflavin with the control group. It was found that when photosensitizers (eosin Y, fluorescein) and riboflavin were added, the experimental group with riboflavin was significantly higher than the control group. The results show that these additives are helpful in generating electricity, and considering the low cost of riboflavin, they can be applied in subsequent industrial production.</p>
                       <p>3) Practical applicatio</p>
                       <p>In order to verify the practical application effect of the microbattery, we connected the microbattery containing the gene pathways of ycel-pncB, NAD-Nadd-NADM and wild-type microbattery with the red LED lamp bead in series on a pathway, respectively. The results showed that the brightness of LED beads in the wild-type group did not change significantly after being connected with the battery, indicating that the wild-type micro battery did not reach the rated operating voltage of LED beads, while the red LED beads in the engineering bacteria group could be used normally and emit bright red light. The results showed that the gene we transferred significantly improved the electricity generation capacity of Shiwanella. And can actually provide electrical energy for some electrical appliances.</p>
                       <h2 class="subtitle" id="list5">   
                        Co-culture system
                    </h2> 
                    <p>4.3.1 Co-culture growth</p>
                       <p>We attempted to combine the BG11 medium required for cyanobacteria culture with the M9 buffer used by Shiwanella to generate electricity and configured MBG11 medium to achieve the growth of two kinds of bacteria through the same medium, so as to prepare for the subsequent continuous culture experiment.</p>
                       <p>We used MBG11 culture medium to culture cystidia and Shiwanella respectively, and measured OD values of cystidia culture medium at 730nm and Shiwanella culture medium at 600nm. The OD value is measured every 24 hours, and a growth curve is obtained as follows. It can be seen that MBG11 culture can make the normal growth of cystidia. However, at the same time, we found that Shewanella could grow at present, but the growth rate was slow. When the bacteria reached the stable stage, the concentration of the bacteria was not high, and the growth situation was far less than that cultured in LB medium. We analyzed that the reason for this result might be the lack of nutrients. At the same time, we also noticed that when M9 buffer was used to generate electricity for a long time, the bacterial concentration in the device was almost unchanged, indicating that in a low-nutrient environment, Shewanella could only maintain survival and generate electricity but could not grow and multiply. In response to this situation. We will further improve the medium system in the future, and at the same time increase the concentration of Shewanella bacteria in LB medium in advance and add MBG11 medium to produce electricity.</p>
                       <p>4.3.2 Co-culture to produce electricity</p>
                       <p>We used MBG11 medium to measure the electric production performance of Shiwanella in logarithmic growth phase. The results are as follows, indicating that it can produce electricity normally in MBG11 medium containing lactic acid, and further prove the feasibility of co-culture scheme.

</p>



                </article> 
            </div>
        </div>
    </div>
</div>
    <div class="footerdivide"></div>
<a href="javascript:scrollTo(0, 1000000)"><img src="https://2022.igem.wiki/hust-china/static/arrow-down-square.svg" alt="arrow-down-square"
      id="to-the-bottom"></a>
  <a href="javascript:scrollTo(0, 0)"><img src="https://2022.igem.wiki/hust-china/static/arrow-up-square.svg" alt="arrow-up-square" id="to-the-top"></a>
<script>
    var herfsvg = document.getElementById("herfsvg")
    herfsvg.style.left = "48%";
    herfsvg.style.top = "90%";
    var titlebg = document.getElementById("titlebg")
    titlebg.style.height = (window.screen.height - 100)   + "px"//"200px"
    window.onscroll = function () {
      var currentHeight =
        document.documentElement.scrollTop ||
        window.pageYOffset ||
        document.body.scrollTop;
      // 页面滚动超过300px就显示
      if (currentHeight > 300) {
        document.getElementById('to-the-top').style.display = 'block'
      } else {
        document.getElementById('to-the-top').style.display = 'none'
      }
      if (currentHeight < 900000) {
        document.getElementById('to-the-bottom').style.display = 'block'
      } else {
        document.getElementById('to-the-bottom').style.display = 'none'
      }
    }
  </script>


    <!--  Video  -->
  </div>

  <!-- Footer: MUST mention license AND have a link to team wiki's repository on gitlab.igem.org -->
  <footer class="footer">
</footer>

  <!-- Wiki Tools: Teams are allowed to remove it -->
  <style>
    .wiki-tools-footer {
        --bs-gutter-x: 1.5rem;
        background: #222124;
        margin: 0;
        max-width: 100%;
        padding: 1rem 3rem;
        display: flex;
        gap: 1.5em;
    }
    .wiki-tools-footer a {
        text-decoration: none;
        color: white;
        font-weight: 400;
        transition: color 0.3s;
    }
    .wiki-tools-footer a:hover {
        color: #08C784;
    }
    .wiki-tools-footer img {
        object-fit: contain;
    }
</style>

<section class="wiki-tools-footer container">
    <a id="wiki-tools-link">Access Wiki Tools</a>
    <a id="wiki-tools-repo">View Source Code</a>
    <img id="wiki-tools-pipeline">
</section>

<script>
    const wikiToolsLink = document.getElementById('wiki-tools-link')
    const wikiToolsRepo = document.getElementById('wiki-tools-repo')
    const wikiToolsPipeline = document.getElementById('wiki-tools-pipeline')

    const path = document.location.pathname.split('/')

    const competitionSlug = document.location.host.split('.')[0]
    const teamSlug = path[1]
    const page = path[2] === '' ? 'index' : path[2]

    wikiToolsLink.href = `https://tools.igem.org/wiki-tools/${competitionSlug}/${teamSlug}/${page}`
    wikiToolsRepo.href = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}`
    wikiToolsPipeline.src = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}/badges/main/pipeline.svg`
</script>

  <!-- Bootstrap Bundle with Popper -->
  <script src="static/bootstrap.bundle.min.js"></script>
  </body>
  
</html>
