<head>
  <meta charset="utf-8">
  <title>Results | HUST-China-iGEM 2023</title>
  <meta content="width=device-width, initial-scale=1" name="viewport">
  <meta content="Webflow" name="generator">
  <link href="static/normalize.css" rel="stylesheet" type="text/css">
  <link href="static/webflow.css" rel="stylesheet" type="text/css">
  <link href="static/igem-hust-china-2023.other_pages.css" rel="stylesheet" type="text/css">
  <link href="https://fonts.googleapis.com" rel="preconnect">
  <link rel="license" href="https://creativecommons.org/licenses/by/4.0/" />
  <link href="https://fonts.gstatic.com" rel="preconnect" crossorigin="anonymous">
  <script src="static/jquery.js" type="text/javascript"></script>
  <script src="static/webfont.js" type="text/javascript"></script>
  <script src="static/webflow.js" type="text/javascript"></script>
  <script type="text/javascript">
    WebFont.load({
      google: {
        families: ["Inconsolata:400,700", "Montserrat:100,100italic,200,200italic,300,300italic,400,400italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic", "Droid Serif:400,400italic,700,700italic", "Oswald:200,300,400,500,600,700", "Merriweather:300,300italic,400,400italic,700,700italic,900,900italic"]
      }
    });
  </script>
  <script type="text/javascript">
    ! function (o, c) {
      var n = c.documentElement,
        t = " w-mod-";
      n.className += t + "js", ("ontouchstart" in o || o.DocumentTouch && c instanceof DocumentTouch) && (n.className += t + "touch")
    }(window, document);
  </script>
  <link href="images/favicon.ico" rel="shortcut icon" type="image/x-icon">
  <link href="images/webclip.png" rel="apple-touch-icon">


<style>
    #subtitle1:target {
        padding-top: 60px;
        margin-top: -60px;
    }


    #to-the-top {
        /*svg导航*/
        height: 30px;
        width: 30px;
        bottom: 150px;
        right: 0px;
        position: fixed;
        display: none;
        /*padding: 10px;*/
        margin: 10px;
    }

    #to-the-bottom {
        height: 30px;
        width: 30px;
        bottom: 110px;
        right: 0px;
        position: fixed;
        /*padding: 10px;*/
        margin: 10px;
    }
    body{
      margin=0;
    }
</style>

<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="https://static.igem.wiki/common/icons/favicons/igem-2022.svg"/>
    <link rel="license" href="https://creativecommons.org/licenses/by/4.0/"/>

    <!-- Bootstrap CSS -->
    <link href="static/bootstrap.min.css" rel="stylesheet">

    <!-- JQuery Js -->
    <script type="text/javascript" src="static/jquery.js")}}" ></script>
    <script type="text/javascript" src="static/jqueryrotate.js")}}" ></script>


    <!-- Custom CSS -->
    <link href="static/style.css" rel="stylesheet">

    <!-- Font style-->
    
    <title> | HUST-China - iGEM 2022</title>
    <style>
      @font-face {
	font-family:Silkscreen;
	src: url(static/Silkscreen-Regular.ttf);
}
@font-face {
    font-family: Montserrat;
    src: url(static/Montserrat-Regular.ttf);
}

@font-face {
    font-family: Mukta;
    src: url(static/Mukta-Regular.ttf);
}

@font-face {
    font-family: Ubuntu;
    src: url(static/Ubuntu-Regular.ttf);
}

@font-face {
    font-family: Josefin_Sans;
    src: url(static/JosefinSans-Regular.ttf);
}

      </style>
    
  </head>
<body class="body">
<div data-animation="default" class="navbar-logo-left-container shadow-three w-nav" data-easing2="ease-out"
    data-easing="ease-in" data-collapse="all" role="banner" data-no-scroll="1" data-duration="400" id="page-header"
    data-doc-height="1">
    <div class="navbar-wrapper">
      <div class="w-layout-blockcontainer container-2 w-container">
        <img src="https://static.igem.wiki/teams/4595/wiki/home/logo.png" loading="lazy" alt="" class="team-logo"></div>
      <nav role="navigation" class="nav-menu-wrapper w-nav-menu">
        <ul role="list" class="nav-menu-two w-list-unstyled">
          <li>
            <a href="index.html" target="_self" aria-current="page" class="nav-link w--current">Home</a>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1850" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1851" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div class="text-block-7"><span>Project</span></div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="description"
                  class="nav-dropdown-link w-dropdown-link">Description</a>
                <a href="safety" class="nav-dropdown-link w-dropdown-link">Safety</a>
                 <a href="implementation"
                  class="nav-dropdown-link w-dropdown-link">Proposed<br>Implementation</a>

              </nav>
            </div>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="f73ff7f3-a811-76cc-69d5-a6519c08fec2" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="f73ff7f3-a811-76cc-69d5-a6519c08fec3" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div class="text-block-7">Wet Lab</div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="design" class="nav-dropdown-link w-dropdown-link">Design</a>
                <a href="engineering"
                  class="nav-dropdown-link w-dropdown-link">Engineering</a>
                <a href="protocol" class="nav-dropdown-link w-dropdown-link">Protocol</a>
                <a href="results" class="nav-dropdown-link w-dropdown-link">Results</a>
                <a href="notebook" class="nav-dropdown-link w-dropdown-link">Notebook</a>
                <a href="parts" class="nav-dropdown-link w-dropdown-link">Parts</a>
              </nav>
            </div>
          </li>
          <li>
            <a href="hardware" class="nav-link">Hardware</a>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1869" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f186a" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div>HP</div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="human-practices"
                  class="nav-dropdown-link w-dropdown-link">Human<br>Practices</a>
                <a href="education"
                  class="nav-dropdown-link w-dropdown-link">Education&amp;<br>Communication</a>
              </nav>
            </div>
          </li>
          <li>
            <div data-hover="true" data-delay="0" class="nav-dropdown w-dropdown">
              <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1845" class="nav-dropdown-toggle w-dropdown-toggle">
                <div data-w-id="ea067c3b-f4d1-2ea5-a13c-4a780c5f1846" class="nav-dropdown-icon w-icon-dropdown-toggle">
                </div>
                <div>Team</div>
              </div>
              <nav class="nav-dropdown-list shadow-three mobile-shadow-hide w-dropdown-list">
                <a href="contribution"
                  class="nav-dropdown-link w-dropdown-link">Contribution</a>
                <a href="members" class="nav-dropdown-link w-dropdown-link">Members</a>
                <a href="attributions"
                  class="nav-dropdown-link w-dropdown-link">Attributions</a>
              </nav>
            </div>
          </li>
        </ul>
      </nav>
      <div class="menu-button w-nav-button">
        <div class="icon w-icon-nav-menu"></div>
      </div>
    </div>
</div>

  
  <!-- Header -->
    
<header>

<a href="#sectitle1"><img src="https://2022.igem.wiki/hust-china/static/arrow-bar-down.svg" alt="" id="herfsvg"></a>
<a href="#sectitle1">
  <div id="titlebg" style="background: url(https://static.igem.wiki/teams/4595/wiki/results/results/results.jpg); background-size:cover"></div>
  </a>
</header>

  <!-- Page Content -->
  <div class="container">
    
 <div class="container">
        <div class="catalogue">
            <h4>content</h4>
            <ul>
                <li><a href="#sectitle1">Introduction</a></li>
                <li><a href="#sectitle2">Plasmid amplification and reconstruction </a></li>
                <ul>
                  <li><a href="#list1"> Plasmids for <i>Synechocystis</i> PCC6803 </a></li>
                  <li><a href="#list2"> Plasmids for <i>S. oneidensis</i> MR-1</a> </li>
                </ul>
                <li><a href="#sectitle3">Strain construction</a></li>
                
                <li><a href="#sectitle4">Expression Validation</a></li>
                <ul>
                  <li><a href="#list3"> Synechocystis PCC6803</a></li>
                  <li><a href="#list4"> <i>S. oneidensis</i> MR-1</a></li>
                  <li><a href="#list5"> Co-culture system</a></li>
                </ul>
            </ul>
        </div>
    <div class="main content">
        <div class="row">
            <div class="col-lg-1"></div>
            <div class="content col-lg-10">
                <article>
                  <h1 class="sectitle title-cover" id="sectitle1">Section 1 : Introduction</h1>
                        <p>Our project aimed to enable <i>Synechocystis</i> PCC6803 to produce lactic acid at a high efficiency, enhance the electricity producing capacity of <i>S. oneidensis</i> MR-1, as well as improve the resistance of both bacteria. In order to achieve these goals, we construct mainly 4 plasmids: pLactate and pResistence6803 for <i>Synechocystis</i> PCC6803, pElectricity and pResistance MR-1 for <i>S. oneidensis</i> MR-1. Besides 4 main plasmids, we also constructed 3 plasmids in order to test the function of genes more conveniently. </p>
                         <p>We transferred them into Synechocystis PCC6803 and <i>S. oneidensis</i> MR-1 for expressing the proteins we needed. At the same time, in order to verify the efficiency of electricity production, we also carried out and verified our project from all aspects.</p>
                    <h1 class="sectitle title-cover" id="sectitle2">Section 2 : Plasmid amplification and reconstruction </h1>
                     <p>We first transformed the plasmids containing designed sequence synthesized by the company into  Escherichia coli Top10 for amplification, so we have enough amount of plasmid to carry out subsequent experiments. Then colony PCR showed a successful transformation, and required plasmids were extracted for the next experiment. </p>
                      <p>We then obtained target fragments by means of PCR, and then constructed them into the plasmids we need. </p>
                    <h2 class="subtitle" id="list1">2.1 Plasmids for <i>Synechocystis</i> PCC6803 </h2>
                        <p>We have 2 plasmids, pLactate and pResistance6803 for <i>Synechocystis</i> PCC6803. We verified the size of each plasmids as well as all the fragments involved in constructing the plasmids. In addition to pLactate and pResistance6803, we also constructed plasmid Omcs to ensure the function of it individually.</p>
                        <p>pLactate includes LdhA enabling <i>Synechocystis</i> PCC6803 to produce lactic acid by means of photosynthesis, Lldp to transform intracellular lactate to the extracellular as well as Omcs improving the efficiency of electricity transformation.</p>
                         <img src="https://static.igem.wiki/teams/4595/wiki/results/results/gaitu.png" alt="" class="normimg1" >
                        <img src="https://static.igem.wiki/teams/4595/wiki/results/results/1-gh.png" alt="" class="normimg1" >
                        <i class="imgexplain">Fig 1. The PCR result of Pcpc560-ldhA-lldP-Tpsbc-Ppsba2-omcs-Tpsbc.</i>
                        <p>The bands of Pcpc560-ldhA-lldP-Tpsbc-Ppsba2-omcs-Tpsbc(5000+bp) from PCR are identical to the theoretical lengths of 5479bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained.</p>

                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/2-g-1.png" alt="" class="normimg">
                       <i class="imgexplain">Fig 2. The PCR result of Pcpc560-ldhA-lldP-Tpsbc.</i>
                       <p>The bands of Pcpc560-ldhA-lldP-Tpsbc(3000+bp) from PCR are identical to the theoretical lengths of 3643bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had successfully been obtained.</p>
                     <img src="https://static.igem.wiki/teams/4595/wiki/results/results/3-1-g-2-g-2.png" alt="" class="normimg">
                     <img src="https://static.igem.wiki/teams/4595/wiki/results/results/3-2-g-2-gh.png" alt="" class="normimg">
                       <i class="imgexplain">Fig 3. The PCR result of Ppsba2-omcs-Tpsbc from pLactate(left) and Omcs(right). </i>
                     <p>The bands of Ppsba2-omcs-Tpsbc(almost 2000bp) from PCR are identical to the theoretical lengths of 1866bp(from pLactate) and 1820bp(from Omcs) estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had been successfully obtained.</p>
                      <img src="https://static.igem.wiki/teams/4595/wiki/results/results/0168wz.png" alt="" class="normimg1">
                       <i class="imgexplain">Fig 4. The PCR result of PUC57-slr0168(vector).</i>
                        <p>The bands of PUC57-slr0168(vector)(almost 5000bp) from PCR are identical to the theoretical lengths of 4727bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had been successfully obtained.</p>
                         <p>All the results above showed us a success of the construction of pLactate and Omcs.</p>
                         <p>pResistance6803 is constructed to enhance the resistance of <i>Synechocystis</i> PCC6803 under adversities, especially the antioxidant capacity and the ability to protect DNA from damages. </p>
                         <img src="https://static.igem.wiki/teams/4595/wiki/results/results/fig2.png" alt="" class="normimg1">
                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/5-lir.png" alt="" class="normimg1">
                       <i class="imgexplain">Fig 5. The PCR result of fragment PrbcL-dsup-Tpsbc-PrbcL-gshA-gshB-Tpsbc.</i>
                       <p>The bands of PrbcL-dsup-Tpsbc-PrbcL-gshA-gshB-Tpsbc(almost 5000bp) from PCR are identical to the theoretical lengths of 4746 estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had been successfully obtained. </p>
                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/6-l-1.png" alt="" class="normimg">
                       <i class="imgexplain">Fig 6. The PCR result of fragment PrbcL-dsup-Tpsbc.</i>
                       <p>The bands of PrbcL-dsup-Tpsbc	(almost 2000bp) from PCR are identical to the theoretical lengths of 1848 estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had been successfully obtained. </p>
                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/7-l-2.png" alt="" class="normimg">
                       <i class="imgexplain">Fig 7. The PCR result of fragment PrbcL-gshA-gshB-Tpsbc.</i>
                       <p>The bands of PrbcL-gshA-gshB-Tpsbc(almost 3000bp) from PCR are identical to the theoretical lengths of 2889 estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had been successfully obtained. </p>
                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/2030wz.png" alt="" class="normimg1">
                       <i class="imgexplain">Fig 8. The PCR result of fragment PUC57-slr2030.</i>
                       <p>The bands of PUC57-slr2030(almost 5000bp) from PCR are identical to the theoretical lengths of 4633 estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had been successfully obtained. </p>
                       <p>All the results above showed us a success of the construction of pResistance6803. </p>
                    <h2 class="subtitle" id="list2">   
                        2.2 Plasmids for <i>S. oneidensis</i> MR-1
                    </h2> 
                        <p>We have 2 plasmids, pElectrcity and pResistance MR-1 for <i>S. oneidensis</i> MR-1. We verified the size of each plasmids as well as all the fragments involved in constructing the plasmids . Besides pElectricity and pResistanceMR-1, we also constructed plasmids to ensure the function of ycel-pncB and nadE-nadD-nadM individually.</p>
                        <p>pElectricty is constructed with ycel-pncB and nadE-nadD-nadM, both can elevate the electricity production rate of <i>S. oneidensis</i> MR-1. </p>
                        <img src="https://static.igem.wiki/teams/4595/wiki/results/results/fig3.png" alt="" class="normimg1">
                        <img src="https://static.igem.wiki/teams/4595/wiki/results/results/9-cd.png" alt="" class="normimg1">
                         <i class="imgexplain">Fig 9. The PCR result of fragment Ptac-ycel-pncB-TrrnB T1-Ptac-nadE-nadD-nadM-TrrnB T1.</i>
                        <p>The bands of Ptac-ycel-pncB-TrrnB T1-Ptac-nadE-nadD-nadM-TrrnB T1(5000+bp) from PCR are identical to the theoretical lengths of 5736 estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had been successfully obtained.</p>
                         <img src="https://static.igem.wiki/teams/4595/wiki/results/results/gaitu2.png" alt="" class="normimg">
                         <i class="imgexplain">Fig 10. The PCR results of Ptac-ycel-pncB-TrrnB T1 and Ptac-nadE-nadD-nadM-TrrnB T1 from pElectricity.</i>
                          <p>The bands of Ptac-ycel-pncB-TrrnB T1(2500+bp) and Ptac-nadE-nadD-nadM-TrrnB T1(about3000 bp) from PCR are identical to the theoretical lengths of 2601bp and 2820bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had been successfully obtained.</p>
                          <img src="https://static.igem.wiki/teams/4595/wiki/results/results/11-c-1.png" alt="" class="normimg">
                         <i class="imgexplain">Fig 11. The PCR results of fragment Ycel-PncB from ycel-pncB.</i>
                          <p>The bands of Ycel-PncB(about 2500bp) from PCR are identical to the theoretical lengths of 2622bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had been successfully obtained.</p>
                          <img src="https://static.igem.wiki/teams/4595/wiki/results/results/12-c-2.png" alt="" class="normimg">
                         <i class="imgexplain">Fig 12. The PCR results of Ptac-nadE-nadD-nadM-TrrnB T1 from nadE-nadD-nadM.</i>
                          <p>The bands of Ptac-nadE-nadD-nadM-TrrnB T1 (about 3000bp) from PCR are identical to the theoretical lengths of 2820bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had been successfully obtained.</p>
                           <img src="https://static.igem.wiki/teams/4595/wiki/results/results/pyydtwz.png" alt="" class="normimg1">
                         <i class="imgexplain">Fig 13. The PCR results of PYYDT(vector).</i>
                          <p>The band of PYYDT(vector)	(5000+bp) from PCR is identical to the theoretical lengths of 5886bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these plasmids had been successfully obtained.</p>
                          <p>All the results above showed us a success of the construction of pElecteicity, ycel-pncB and nadE-nadD-nadM.</p>
                          <p>pResistance MR-1 is constructed with dsup-sodA and oprf, which can lower the level of ROS in <i>S. oneidensis</i> MR-1 and avoid DNA damages, and the latter can assist in the entry and exit of riboflavin and improves the efficiency of electron transfer  </p>
                        <img src="https://static.igem.wiki/teams/4595/wiki/results/results/fig4.png" alt="" class="normimg1">
                        <img src="https://static.igem.wiki/teams/4595/wiki/results/results/14-sir.png" alt="" class="normimg">
                         <i class="imgexplain">Fig 14. The PCR result of Ptac-dsup-sodA-oprf-TrrnB T1.</i>
                         <p>The bands of Ptac-dsup-sodA-oprf-TrrnB T1(3000+bp) from PCR are identical to the theoretical lengths of 3291bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these fragments had been successfully obtained.</p>
                           <img src="https://static.igem.wiki/teams/4595/wiki/results/results/15-s-12.png" alt="" class="normimg">
                         <i class="imgexplain">Fig 15. The PCR result of fragment Ptac-dsup-sodA-TrrnB T1.</i>
                         <p>The bands of Ptac-dsup-sodA-oprf-TrrnB T1(3000+bp) from PCR are identical to the theoretical lengths of 3291bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these fragments had been successfully obtained.</p>
                           <img src="https://static.igem.wiki/teams/4595/wiki/results/results/16-s-3.png" alt="" class="normimg">
                         <i class="imgexplain">Fig 16. The PCR result of fragment Ptac-oprf-TrrnB T1.</i>
                          <p>The bands of Ptac-oprf-TrrnB T1(1000+bp) from PCR are identical to the theoretical lengths of 1135bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these fragments had been successfully obtained.</p>
                          <p>The results above could demonstrate that these plasmids are correctly constructed, so we can transform pLactate, pResistence6803,and omcs into <i>Synechocystis</i> PCC6803, while pElectricity, ycel-pncB, nadD-nadE-nadM and pResistance MR-1into <i>S. oneidensis</i> MR-1 for verification.   </p>
                           
                     <h1 class="sectitle title-cover" id="sectitle3">Section 3 :Strain construction</h1>
                      <p>We have successfully constructed all the target plasmids needed in the four loops, and this step really facilitates our subsequent experiments. At the same time, in order to facilitate the measurement of the function of some elements, we separately constructed plasmids (omcs, ycel-pncB, nadD-nadE-nadM) for some essential elements.</p>
                      <img src="https://static.igem.wiki/teams/4595/wiki/results/results/results1.png" alt="" class="normimg1">
                         <i class="imgexplain">Fig 17. Colony PCR results of pLactate, Omcs and pResistence6803.</i>
                         <img src="https://static.igem.wiki/teams/4595/wiki/results/results/resultsjt2.png" alt="" class="normimg1">
                         <i class="imgexplain">Fig 18. Colony PCR results of target fragments in plasmids pElectricity, ycel-pncB, nadD-nadE-nadM and pResistenceMR-1.</i>
                       <p>All the bands are identical to the theoretical sizes, demonstrating that these plasimds had been successfully transformed and verified.The results are as follows: PLactate: Pcpc560 - ldhA - lldP - Tpsbc - Ppsba2 - omcs -Tpsbc about 5400 bp , pResistance6803: PrbcL - dsup - Tpsbc - PrbcL - gshA - gshB -Tpsbc about 4100 bp , omcs: Ppsba2-omcs-Tpsbc about 1800bp, pElectricty:Ptac-ycel-pncB-TrrnB T1-Ptac-nadD-nadE-nadM-TrrNB T1 about 5700bp, pResistance MR-1: Ptac-dsup-soda-oprf-TrrnB T1 about 3200bp , ycel-pncB: Ptac-Ycel-PNCB-TrrnB T1 about 2500bp, nadD-nadE-nadM: Ptac-nadD-nadE-nadM-TrrNB T1 about 2800bp</p>
                       <p>The theoretical length of 5461bp, 4107bp,1809bp, 5666bp, 3229bp,  2564bp and 2820bp estimated by the design primers location (promoter to terminator) was the same, indicating that we successfully obtained the plasmid we needed.</p>
                       <p>The successful plasmids transformed into <i>Synechocystis</i> PCC6803 and <i>S.oneidensis MR-1</i> was verified by colony PCR.</p>
                       <h1 class="sectitle title-cover" id="sectitle4">Section 4 : Expression Validation </h1>
                       <h3 class="subtitle" id="list3">   
                      4.1 <i>Synechocystis</i> PCC6803
                    </h3> 
                     <h5>4.1.1  Growth curve of <i>Synechocystis</i> PCC6803</h5>
                       <p>We used BG11 to culture the engineered bacteria <i>Synechocystis</i> PCC6803(omcs) and wild type respectively, and measured the OD<sub>730</sub> every 24 hours. A growth curve is obtained as follows.</p> 
                       <p>The results showed that compared to <i>Synechocystis</i> PCC6803,<i>Synechocystis</i> PCC6803(omcs) exhibited a significant improvement in cell concentration at all stages, and there was no significant decrease in growth rate. This suggests that omcs can enhance the photosynthetic efficiency of PSI by increasing ATP synthesis, which promots better growth of cells</p>
                        <img src="https://static.igem.wiki/teams/4595/wiki/engineering/engineering/fig11.png" alt="" class="normimg">
                                    <i class="imgexplain">Fig 19. The growth curve of Synechocystis PCC6803 and Synechocystis PCC6803(omcs) in BG11.</i>
                       <h5>4.1.2 Lactate production verification</h5>
                       <p>We want to co-culture <i>Synechocystis</i> PCC6803 and <i>S. oneidensis</i> MR-1. The nutrient of <i>S.oneidensis MR-1</i> is lactate produced by <i>Synechocystis</i> PCC6803. We tried to determine the production of lactate by HPLC to test the feasibility of the co-culture method.</p>
                      <p>We first measured the standard curve of lactate concentration and determined its retention time. The <i>Synechocystis</i> PCC6803(pLactate) cells were broken by freeze-thaw method, and the supernatant after centrifugation was taken for high performance liquid chromatography (HPLC).</p>
                      <p>A peak with almost the same retention time which is 3.932 min as the lactate standard substance was observed on the spectrum, which confirmed the successful expression of lactate in engineered bacteria. The peak area was compared with the standard curve, which can be seen that the lactate yield of our engineering <i>Synechocystis</i> PCC6803 was 7.26mM, indicating that the gene we introduced could produce lactic acid stably and provide nutrition for <i>S. oneidensis</i> MR-1.</p>
                      <p>This result indicated the successful expression of the imported LdhA and LldP genes, and verified the feasibility of the co-culture system.</p>
                         <img src="https://static.igem.wiki/teams/4595/wiki/results/results/fig20.png" alt="" class="normimg1">
                         <i class="imgexplain">Fig 20. The left figure is the HPLC spectrogram of lactate standard product in different concentration. </i>
                           <i class="imgexplain">The right figure is the standard curve of lactate(0-10 mM).
                         </i>
                         <img src="https://static.igem.wiki/teams/4595/wiki/results/results/fig21.png" alt="" class="normimg1">
                           <i class="imgexplain">Fig 21.The figure is the HPLC spectrogram of Synechocystis PCC6803, Synechocystis PCC6803(omcs) and 5mM Lactate standard product. There is a peak at 3.932 min both in Synechocystis PCC6803(omcs) and 5mM Lactate standard product which represent lactate, and Synechocystis PCC6803 don't have this peak. It indicated that Synechocystis PCC6803(omcs) could product lactate while the wild type don't have this ability.
                         </i>
                       <h3 class="subtitle" id="list4">   
                       4.2 <i>S. oneidensis</i> MR-1
                    </h3> 
                     <h3>4.2.1 Determination of NADH/NAD+ concentration</h3>
                       <p>To verify normal gene expression in <i>S. oneidensis</i> MR-1, we measured the NADH/NAD+ content of wild type and engineered strains ycel-pncB and nadD-nadE-nadM, respectively.</p>
                       <p>Compared to the wild type, <i>S. oneidensis</i> MR-1(ycel-pncB) showed a 22.23% increase in NAD(H/+) amount, while the total amount of NAD(H/+) in <i>S. oneidensis</i> MR-1(nadD-nadE-nadM) increased by 27.34%. This indicates that, <i>S.oneidensis MR-1</i> (ycel-pncB) may contribute to the accelerated transport of niacin and nicotinamide for intracellular NADH synthesis, while <i>S. oneidensis</i> MR-1(nadD-nadE-nadM) facilitates more efficient electron transfer.</p>
                       <img src="https://static.igem.wiki/teams/4595/wiki/engineering/engineering/fig15.jpg" alt="" width="50%">
                                    <i class="imgexplain">Fig 22. The NAD(H/+) concentration of S.oneidensis MR-1, S.oneidensis MR-1(ycel-pncB), S.oneidensis MR-1(nadE-nadD-nadM).</i>
                       <h5>4.2.2 Verification of electricity production of <i>S. oneidensis</i> MR-1</h5>
                       <p>We built a microbial fuel cell device to test the electricity-producing performance of<i>S. oneidensis</i> MR-1.</p>
                       
                       <p> We inoculated <i>S. oneidensis</i> MR-1 cultured overnight in 100 mL LB medium at 30℃ and 200 rpm to OD<sub>600</sub> to 0.6-0.8. The cultured cells were collected by centrifuge and re-suspended in 140 mL anoditic solution. 50 μg/mL kana antibiotics and 18mM lactate were added to maintain constant culture conditions and 0.1 mM IPTG inducer to initiate gene expression. The two electrode chambers of MFC are separated by proton exchange membrane, using carbon cloth as cathode and anode electrodes, and then adding 140mL cathode solution to the cathode chamber, using 2kΩ external resistance to close the external circuit.</p>
                       <p>The specific battery structure of a complete MFC is as follows</p>
                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/fig22.jpg" alt="" class="normimg">
                                    <i class="imgexplain">Fig 23. The microbial fuel cell device we built.</i>
                        <p>1) testing the MFC</p>
                       <p>After construction the MFC, we use a digital multimeter to measure the output voltage of microbial fuel cell.</p>
                       <p>We measured the voltage of <i>S. oneidensis</i> MR-1(ycel-pncB), <i>S. oneidensis</i> MR-1(nadD-nadE-nadM) and compared them with the voltage of <i>S. oneidensis</i> MR-1. Measure a set of voltages every 6 hours and compare as follows.</p>
                         <p>The results showed that <i>S. oneidensis</i> MR-1(ycel-pncB) exhibited a significantly higher power output than the wild type before 30 hours, but experienced a noticeable decrease after 30 hours. It is speculated that this could be attributed to the ability of<i>S. oneidensis</i> MR-1(ycel-pncB)  to accelerate the transport of niacin and nicotinamide for intracellular NADH synthesis, resulting in a higher power output before 30 hours.</p>
                         <p>However, due to the absence of exogenous niacin and nicotinamide supplementation, the availability of NADH synthesis precursors decreases, leading to a decline in intracellular NADH concentration and a slower electron transfer rate, ultimately resulting in reduced power generation efficiency.</p>
                         <p>In contrast, nadD-nadE-nadM exhibited significantly higher discharge peak and prolonged high-efficiency discharge duration compared to the wild type. The highest out put voltage was up to 150.7 mV, with a 42.32% increase in the hightest power output .</p>
<img src="https://static.igem.wiki/teams/4595/wiki/results/results/fig24.png" alt="" class="normimg">
                                    <i class="imgexplain">Fig 24. The out put voltage of S.oneidensis MR-1, S.oneidensis MR-1(ycel-pncB), S.oneidensis MR-1(nadD-nadE-nadM) when the anoditic solution is M9 buffer and 18mM lactate.</i>
                       <p>2) exogenous electron carrier</p>
                       <p>In our search for relevant papers, we learned that exogenous electron carrier such as photosensitivities and riboflavin contribute to extracellular electron transfer, thereby improving the efficiency of electricity generation. Therefore, we investigated the effect of addition of photosensitizers (5 μM Eosin Y, 5 μM Fluorescein) and 40 μM riboflavin on electricity production in <i>S. oneidensis</i> MR-1.</p>
                       <p>We use <i>S. oneidensis</i> MR-1(nadD-nadE-nadM) to try getting the highest voltage output by adding exogenous electron carrier including riboflavin and different  photosensitizers. We constructed the MFC device and  observed that riboflavin significantly increased the peak power output by 5.56 times after 20 hours with the highest voltage at 668mV, while the other exogenous electron carrier showed no significant change. This could be attributed to the optimization of electron carriers and enhanced electron transfer efficiency after the addition of riboflavin.</p>
                       <p>The results show that riboflavin is helpful in generating electricity.And considering the low cost of riboflavin, it can be applied in subsequent industrial production.</p>
 <img src="https://static.igem.wiki/teams/4595/wiki/results/results/fig25.png" alt="" class="normimg">
                                    <i class="imgexplain">Fig 25. The out put voltage of S.oneidensis MR-1(nadD-nadE-nadM)with different exogenous electron carrier.</i>
                       <p>3) Practical application</p>
                       <p>In order to verify the practical application effect of the microbattery, we connected the microbattery containing the gene pathways of <i>S. oneidensis</i> MR-1, <i>S. oneidensis</i> MR-1(ycel-pncB), <i>S. oneidensis</i> MR-1(nadD-nadE-nadM)with a red LED bulb , respectively.</p>
                       <p>The results showed that the brightness of LED bulb did not change significantly after being connected with the <i>S. oneidensis</i> MR-1 micro battery, indicating that the <i>S. oneidensis</i> MR-1 micro battery did not reach the operating voltage of LED bulb, while the red LED bulb in the engineering bacteria group could be used normally and emit bright red light. The hightest voltage could be up to 2.1V</p>
                       <p>The results showed that the gene we transferred significantly improved the electricity generation capacity of <i>S. oneidensis</i> MR-1, which can actually provide electrical energy for some electrical appliances.</p>
                         <iframe title="HUST-China: MFC light up the LED bulb (2023) [English]" width="560" height="315" src="https://video.igem.org/videos/embed/2ed9db19-dd7e-4f76-86a4-e9e6cdada601" frameborder="0" allowfullscreen="" sandbox="allow-same-origin allow-scripts allow-popups"></iframe>
                       <h3 class="subtitle" id="list5">   
                       4.3 Co-culture system
                    </h3> 
                    <h5>4.3.1 Co-culture growth</h5>
                       <p>We attempted to combine the BG11 medium required for <i>Synechocystis</i> PCC6803 culture with the M9 buffer used for <i>S. oneidensis</i> MR-1 to generate electricity and configured MBG11 medium to achieve the growth of two kinds of bacteria through the same medium, so as to prepare for the subsequent continuous culture experiment.</p>
                       <p>We used MBG11 culture medium to culture <i>Synechocystis</i> PCC6803 and <i>S. oneidensis</i> MR-1 respectively, and measured OD values of <i>Synechocystis</i> PCC6803 culture medium at 730nm and <i>S. oneidensis</i> MR-1 culture medium at 600nm. The OD value is measured every 24 hours, and a growth curve is obtained as follows. It can be seen that MBG11 culture can make the normal growth of <i>Synechocystis</i>.</p>
                       <p>However, at the same time, we found that <i>S. oneidensis</i> MR-1 could grow at present, but the growth rate was slow. When the bacteria reached the stable stage, the concentration of the bacteria was not high, and the growth situation was far less than that cultured in LB medium. We analyzed that the reason for this result might be the lack of nutrients.</p>
                       <p>At the same time, we also noticed that when M9 buffer was used to generate electricity for a long time, the bacterial concentration in the device was almost unchanged, indicating that in a low-nutrient environment, <i>S. oneidensis</i> MR-1 could only maintain survival and generate electricity but could not grow and multiply.</p>
                       <p>In response to this situation. We will further improve the medium system in the future, and at the same time increase the concentration of <i>S. oneidensis</i> MR-1 in LB medium in advance and add MBG11 medium to produce electricity.</p>
                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/fig26.png" alt="" class="normimg">
                       <i class="imgexplain">Fig 26. The growth curve of Synechocystis PCC6803 and Synechocystis PCC6803(omcs)in MBG11.Both Synechocystis PCC6803 and Synechocystis PCC6803 (omcs) can grow at  fast speed relatively. Compared to Synechocystis PCC6803, Synechocystis PCC6803 (omcs) exhibited a significant improvement in cell concentrationat all stages, with a more stable growth rate. This provides evidence that omcs can also enhance ATP synthesis and promote the photosynthetic efficiency of PS1 in the MBG11 culture environment, resulting in better cell growth.</i>
                       <img src="https://static.igem.wiki/teams/4595/wiki/results/results/fig27.png" alt="" class="normimg">
                       <i class="imgexplain">Fig 27. The growth curve of S.oneidensis MR-1, S.oneidensis MR-1(ycel-pncB), S.oneidensis MR-1(nadD-nadE-nadM) in MBG11.</i>
                       <h5>4.3.2 Co-culture to produce electricity</h5>
                       <p>We used MBG11 medium to measure the electric production performance of Shiwanella in logarithmic growth phase. The results are as follows, indicating that it can produce electricity normally in MBG11 medium containing lactic acid, and further prove the feasibility of co-culture scheme.</p>
                         <img src="https://static.igem.wiki/teams/4595/wiki/results/results/fig28.png" alt="" class="normimg">
                         <i class="imgexplain">Fig 27. The out put voltage of S.oneidensis MR-1, S.oneidensis MR-1(ycel-pncB), S.oneidensis MR-1(nadD-nadE-nadM) in MBG11.





                </article> 
            </div>
        </div>
    </div>
</div>
    <div class="footerdivide"></div>
<a href="javascript:scrollTo(0, 1000000)"><img src="https://2022.igem.wiki/hust-china/static/arrow-down-square.svg" alt="arrow-down-square"
      id="to-the-bottom"></a>
  <a href="javascript:scrollTo(0, 0)"><img src="https://2022.igem.wiki/hust-china/static/arrow-up-square.svg" alt="arrow-up-square" id="to-the-top"></a>
<script>
    var herfsvg = document.getElementById("herfsvg")
    herfsvg.style.left = "48%";
    herfsvg.style.top = "90%";
    var titlebg = document.getElementById("titlebg")
    titlebg.style.height = (window.screen.height - 100)   + "px"//"200px"
    window.onscroll = function () {
      var currentHeight =
        document.documentElement.scrollTop ||
        window.pageYOffset ||
        document.body.scrollTop;
      // 页面滚动超过300px就显示
      if (currentHeight > 300) {
        document.getElementById('to-the-top').style.display = 'block'
      } else {
        document.getElementById('to-the-top').style.display = 'none'
      }
      if (currentHeight < 900000) {
        document.getElementById('to-the-bottom').style.display = 'block'
      } else {
        document.getElementById('to-the-bottom').style.display = 'none'
      }
    }
  </script>


    <!--  Video  -->
  </div>

  <!-- Footer: MUST mention license AND have a link to team wiki's repository on gitlab.igem.org -->
  <footer class="footer">
</footer>

  <!-- Wiki Tools: Teams are allowed to remove it -->
  <style>
    .wiki-tools-footer {
        --bs-gutter-x: 1.5rem;
        background: #222124;
        margin: 0;
        max-width: 100%;
        padding: 1rem 3rem;
        display: flex;
        gap: 1.5em;
    }
    .wiki-tools-footer a {
        text-decoration: none;
        color: white;
        font-weight: 400;
        transition: color 0.3s;
    }
    .wiki-tools-footer a:hover {
        color: #08C784;
    }
    .wiki-tools-footer img {
        object-fit: contain;
    }
</style>

<section class="wiki-tools-footer container">
    <a id="wiki-tools-link">Access Wiki Tools</a>
    <a id="wiki-tools-repo">View Source Code</a>
    <img id="wiki-tools-pipeline">
</section>

<script>
    const wikiToolsLink = document.getElementById('wiki-tools-link')
    const wikiToolsRepo = document.getElementById('wiki-tools-repo')
    const wikiToolsPipeline = document.getElementById('wiki-tools-pipeline')

    const path = document.location.pathname.split('/')

    const competitionSlug = document.location.host.split('.')[0]
    const teamSlug = path[1]
    const page = path[2] === '' ? 'index' : path[2]

    wikiToolsLink.href = `https://tools.igem.org/wiki-tools/${competitionSlug}/${teamSlug}/${page}`
    wikiToolsRepo.href = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}`
    wikiToolsPipeline.src = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}/badges/main/pipeline.svg`
</script>

  <!-- Bootstrap Bundle with Popper -->
  <script src="static/bootstrap.bundle.min.js"></script>
  </body>
  
</html>
